contracts/services/FaucetService.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../shared/IVersioned.sol";
+
+contract FaucetService is IVersioned {
```

Comment:
I will update this contract as per [SD-467](https://linear.app/ampnet/issue/SD-467/write-basic-faucet-contract-for-funding-new-users-wallets).